kura/org.eclipse.kura.wire.provider/OSGI-INF/metatype/org.eclipse.kura.wire.graph.WireGraphService.xml

```diff
@@ -15,7 +15,7 @@
         <AD id="WireGraph"
             name="WireGraph"
             type="String"
-            cardinality="0"
+            cardinality="65535"
```

Comment:
Why this change? You'll get an Array that is now what we originally intended.

Reply:
This is also  due to the the failure of OCD validation.This value must be greater than 1 to pass the OCD validation. I don't know why. I search some documentation, in this case must to set a large positive value. You can boot kura and test this issue so i believe you can fix it.